components/user-info/frontend/src/view/edit/edit-dialog.html

```diff
@@ -0,0 +1,44 @@
+<div class="edit-user-info-form">
+  <p translate>user-info.edit.introduction</p>
+  <form class="form-stacked center-block" name="asyncTaskDialogCtrl.editUserInfoForm" role="form">
+    <div class="form-group">
+      <label class="control-label" for="user-info-form-givenName" translate>
+        user-info.edit.given-name
+      </label>
+      <input autofocus type="text" class="form-control" focusable-input
+        id="user-info-form-givenName" name="name"
+        placeholder="{{ asyncTaskDialogCtrl.context.user.user.name.givenName }}"
```

Comment:
In other 'edit' style forms (edit app, org, space, etc) the input value starts with the current value and the placeholder contains generic help text.